src/app/store/effects/router-effects.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { Title } from '@angular/platform-browser';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { RouterNavigationAction, ROUTER_NAVIGATION } from '@ngrx/router-store';
```

Comment:
I think the `ALL_CAPS` syntax is now deprecated. Try `routerNavigationAction` - you may need to cast payload though.